.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question on use of dotnet sdk
+    url: https://github.com/microsoftgraph/msgraph-sdk-dotnet/discussions
+    about: Please add your question in the discussions section of the repo
+  - name: Question on use of kiota
+    url: https://github.com/microsoft/kiota/discussions
+    about: Please add your question in the discussions section of the repo
```

Comment:
Should we add entries for service feature requests and issues questions?

Reply:
Hi @andrueastman , for service feature requests, I have left the instructions on how to on the service bug report instructions.md file as was. This is because I noticed we have more useful instructions than just pointing people to the service reporting endpoint. We could improve this as we go, but first, I needed to put up the PR for MVP and improve on it as we go.